### DISCUSSION TITLE
Supress pretty print in JSON output

### DISCUSSION THREAD

**Discussion by volodymyrprokopyuk** [mikefarah/yq#1985] [Q&A] [answered]
Hi,

How to suppress pretty printing of JSON output. The `. style="flow"` works for YAML, but with `-o json` the output is indented multi-line JSON. Is it possible to get a single line JSON string?

Thank you!

**Comment by mikefarah** [comment, marked as answer]
Yep, use the `-I, --indent` flag and set it to 0

```
yq -I0 blah.json
```

**Reply by volodymyrprokopyuk** [reply]
It works! Thank you!
